src/hooks/usePolling.ts

```diff
@@ -23,11 +23,12 @@ type usePollingType = {
  * This hook uses determines polling status based on browser status and page visibility.
  * Depending on these values, it calls start and stop polling functions supplied from an
  * Apollo useQuery hook.
- * @param startPolling - Function from useQuery that is called when online & visible
- * @param stopPolling - Function from useQuery that is called when offline or not visible
- * @param refetch - Optional function from useQuery that can be used to refetch data
- * @param shouldPollFaster - Optional boolean to enable increased poll rat.
- * @param initialPollingState - Optional boolean to indicate the initial polling state
+ * @param props - Object containing the following:
+ * @param props.stopPolling - Function from useQuery that is called when offline or not visible
+ * @param props.refetch - Optional function from useQuery that can be used to refetch data.
+ * @param props.shouldPollFaster - Optional boolean to enable increased poll rate.
+ * @param props.initialPollingState - Optional boolean to indicate the initial polling state.
+ * @param props.startPolling - Function from useQuery that is called when online and visible
```

Comment:
do you know why whenever the props comes in as an object, its individual fields are always typed as `any`? I see `any` when I hover over these fields, but I get proper types when the props isn't an object

Reply:
I'm not sure tbh. I think vscode intellisense is just having a tough time parsing these object fields.